pages/project/components/collaborators.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Fragment, useRef } from 'react';
+import { useSelector } from 'react-redux';
+import { Snippet } from '@asl/components';
+
+function RemoveUser({ id }) {
+  const removeUserUrl = useSelector(state => state.static.removeUserUrl);
+  const url = useSelector(state => state.static.url);
+  const formRef = useRef(null);
```

Comment:
What's this for?

Reply:
Oh. It's for hacking a link to submit the form.

Why not use a button? I think there are even styles for making buttons look like links.

Guess: `className="link"`

Reply:
hahaha - that would be too easy! you get a black link, and when you hover it turns into a button.

Reply:
We manage it ok for training modules. 

https://github.com/UKHomeOffice/asl-pages/blob/master/pages/pil/dashboard/views/section-details.jsx#L56

Reply:
ah - I think it's a problem if you use a `Button` not a `button`

Reply:
You have to say it quietly or the CSS gets scared.

Reply:
😆